api/allIncidents/incidentsRouter.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const router = express.Router();
+const Incidents = require('./incidentsModel');
+const { checkIncidentExists } = require('../middleware');
+
+// TODO document shape of objects coming and going
+
+/**
+ * @swagger
+ * /:
+ *  GET:
+ *    Summary: Path returning all approved incidents in reverse chronological order
+ *    tags:
+ *      - incidents
+ *    produces:
+ *      - application/json
+ *    responses:
+ *      200:
+ *        description: Success ... returns an array of incident objects
+ *      500:
+ *        description: Server response error
+ */
+
+router.get('/', (req, res, next) => {
```

Comment:
Does not include latest code in previously submitted PR

Reply:
yeah that change wasn't made until after I made this PR I believe, I'll pull before recommiting